x-pack/test/functional/config.js

```diff
@@ -236,6 +236,39 @@ export default async function ({ readConfigFile }) {
           ],
         },
 
+        global_discover_all: {
+          kibana: [
+            {
+              feature: {
+                discover: ['all'],
+              },
+              spaces: ['*'],
+            },
+          ],
+        },
+
+        global_dashboard_read: {
+          kibana: [
```

Comment:
I guess this role isn't used in this PR?  I guess it's OK to leave it since it might get used in other tests, and it also helps when manually debugging an issue.

Reply:
removed for now.